### DISCUSSION TITLE
Implementation of reverse_loccN / reverse_hiccN opcodes

### DISCUSSION THREAD

**Discussion by mathieugarcia** [sfztools/sfizz#1207] [Requests]
Hi,

The `reverse_loccN` and `reverse_hiccN` opcodes would make a great addition for manipulating the playback direciton of a `region`.

Reference : http://sfzformat.com/opcodes/reverse_loccN

Thanks!

**Comment by paulfd** [comment]
Hi! This is a tricky one honestly. Playing reversed has a lot of implications on the engine. The probable straightforward way to implement would be to duplicate regions where this opcode is present and switch them on/off depending on CC, which is akin to what sfzformat proposes as a workaround

> Not implemented in ARIA, but an alternative is to use two regions, one with direction=reverse then switch region with [loccN / hiccN](http://sfzformat.com/opcodes/loccN).

Besides convenience is there anything preventing you from doing this?

**Comment by mathieugarcia** [comment]
Hey @paulfd !

Thanks for your prompt reply.

No, not really, it was purely for convenience and to keep the SFZ mapping as clean as possible.

In my scenario, the SFZ is built and updated as the user programs samples, modulations, filters, etc. and manipulate them dynamically (for live play).

Using two region is an acceptable solution, although implementing this opcode would be a welcomed addition in sfizz!
